### PR TITLE
Add chip

### PR DESCRIPTION
**Pull request by samjett247**
- Made timeseries datapoints not connect
- Added chip material-ui element to show the level of the course
- Added tooltip for more detailed info
- Styled things a bit

Feel free to change anything up as desired. Colorschemes, placements, font sizes, etc.

### CODE REVIEW

**Comment by schuermannator** [comment]
Nice, I'll give it a look here in a bit

**Comment by samjett247** [comment]
Hey just wanted to give a quick update on whats going on here. In this PR I:
- Added a chip to the search by instructor, showing the typical enrollment age
- Tried to make the dataview mobile compatible. I think it looks pretty good, not perfect by any means but pretty good. Did this through a combination of css and some React-specific mobile stylings, especially for ChartJS where I couldnt specify parameters in em, vw, or another responsive unit
- Added Click-Between functionality to allow the visitors to easily move between the courses and instructors, at least in Table 1.
- Changed the styling of Timeseries figures to just be a bar chart and specific points for specific instructors/courses. I think it is marginally more comprehensible and better than before.

Thats pretty much it. I know its a lot of files changed, just let me know if you want to discuss reasons or practices, might be a good idea.

File: public/index.html [hunk 1]
```diff
@@ -50,5 +50,6 @@
       You can add webfonts, meta tags, or analytics to this file.
       The build step will place the bundled scripts into the <body> tag.
     -->
+    <div id="div" style="height:0;width:0;outline:none;border:none;padding:none;margin:none;"></div>
```
Comment (samjett247) [inline public/index.html]: This was a workaround to get an idea of the window size in React.

File: src/App.css [hunk 2]
```diff
@@ -6,7 +6,7 @@
```
Comment (samjett247) [inline src/App.css]: Might be a good idea to organize this or separate into files. Just a lot of stuff.

File: src/components/Landing.js [hunk 3]
```diff
@@ -5,13 +5,14 @@ import { connect } from 'react-redux';
...
+    // const valid_search = true // For debugging
     if (!valid_search) {
       return (
         <div className="App">
           <div className="Info">
-            <p>
+            <p style={{fontSize:'0.8em'}}>
               {' '}
-              Disclaimer: This website is not affiliated with nor approved by
+              <b>Disclaimer</b>: This website is not affiliated with nor approved by
```
Comment (samjett247) [inline src/components/Landing.js]: Sorry bout the inline styling i know that a no-no

File: src/components/MobileTools.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (samjett247) [inline src/components/MobileTools.js]: These are just tools to get window sizes in React and if mobile or not in react

File: src/components/SearchAutocomplete.js [hunk 5]
```diff
@@ -8,46 +8,48 @@ import Typography from '@material-ui/core/Typography';
 import TextField from '@material-ui/core/TextField';
```
Comment (samjett247) [inline src/components/SearchAutocomplete.js]: I just trimmed down this file. A lot of things were never used.

Comment (schuermannator) [reply src/components/SearchAutocomplete.js]: kk

File: src/components/SearchAutocomplete.js [hunk 6]
```diff
@@ -8,46 +8,48 @@ import Typography from '@material-ui/core/Typography';
...
     position: 'absolute',
-    left: 2,
-    fontSize: 16,
+    left: 1,
+    fontSize: '0.9em',
+  },
+  options: {
+    fontSize: '0.9em',
+    display: 'flex',
+    /// Lol this stupid ass thing wont wrap the text to the next line
```
Comment (schuermannator) [inline src/components/SearchAutocomplete.js]: lamo

File: update_course_instructor_lists.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
+"""
```
Comment (schuermannator) [inline update_course_instructor_lists.py]: cache in Redis -- also move to shared

**Pull request merged**